src/envo/envo-edit.owl

```diff
@@ -40086,6 +40120,21 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105010> "plastic pellet"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105010> <http://purl.obolibrary.org/obo/ENVO_01000776>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105016> (foamed plastics)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://doi.org/10.1016/B978-185617431-2/50011-6>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://doi.org/10.1533/9780857093967.2.309>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105016> "A polymer which has a cellular structure that comprises a solid polymer matrix and voids containing air phase."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105016> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105016> <https://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105016> <http://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105016> "2021-05-07T12:26:54Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105016> "cellular polymers"@en)
```

Comment:
Careful, other polymers could also be cellular etc - consider broad synonyms here.